Install.sbt

```diff
@@ -0,0 +1,28 @@
+import sbt.{Def, Keys}
+import Keys.`package`
+import sbt.complete.Parsers._
+
+lazy val installToNetLogo = Def.inputKey[Unit]("Install To NetLogo (Mac OS X only)")
+
+installToNetLogo := {
+  val version: String = (Space ~> StringBasic).parsed
+  val netLogoDir = file(s"/Applications/NetLogo $version/")
+  if (netLogoDir.exists && netLogoDir.isDirectory) {
+    val createdFile = (Keys.`package` in Compile).value
+    val installationDir = netLogoDir / "extensions" / "xw" / "widgets" / "LSWidgets"
+    if (! installationDir.isDirectory) {
+      installationDir.mkdirs()
+      IO.copy(
+        Seq("miglayout-core", "miglayout-swing").map(
+          p => (baseDirectory.value / s"$p-4.2.jar", installationDir / s"$p-4.2.jar")))
+    }
+    val installationFile = installationDir / createdFile.getName
+    if (installationFile.exists) {
+      IO.delete(installationFile)
+    }
+    IO.copy(Seq((createdFile -> installationFile)))
+    println(s"latest LSWidgets installed to ${installationFile.getPath}")
+  } else {
+    println(s"could not find /Applications/NetLogo $version/")
+  }
+}
```

Comment:
It must definitely works for me. You need to symlink from the xw's `widgets` directory to this project's root.

```
▶ pwd
/Users/bryan/Development/NetLogo/eXtraWidgets/xw/widgets

▶ ls -l LSWidgets
lrwxr-xr-x  1 bryan  staff    54B Mar 31 12:46 LSWidgets -> /Users/bryan/Development/NetLogo/xw-widgets/ls-widgets
```